extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -155,6 +160,43 @@ require(
 			tracker.track(trackingParams);
 		}
 
+		function getWIYLRequestSentKey() {
+			return 'wikiaInYourLangRequestSent' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLNotificationShownKey() {
+			return 'wikiaInYourLangNotificationShown' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLMessageKey() {
+			return targetLanguage + 'WikiaInYourLangMessage' + retrieveLinkTitle() + cacheVersion;
+		}
+
+		function getWIYLLinkTitlesKey() {
+			return targetLanguage + 'WikiaInYourLangLinkTitles' + cacheVersion;
+		}
+
+		function saveLinkTitle(linkAddress) {
+			var linkAddressAry = linkAddress.split('/'),
+				listOfCachedTitles = '',
+				linkTitle = linkAddressAry[linkAddressAry.length - 1].length === 0 ? 'main' : linkAddressAry[linkAddressAry.length - 1];
+
+			listOfCachedTitles = cache.get(getWIYLLinkTitlesKey());
+			if ( listOfCachedTitles ) {
+				listOfCachedTitles[articleTitle] = linkTitle;
+			} else {
+				listOfCachedTitles = {};
+				listOfCachedTitles[articleTitle] = linkTitle;
```

Comment:
Rather than doing `listOfCachedTitles[articleTitle] = linkTitle;` twice, this could be simplified to something like:
```javascript
if (!listOfCachedTitles) {
	listOfCachedTitles = {};
}

listOfCachedTitles[articleTitle] = linkTitle;
```